Descent3/args.cpp

```diff
@@ -114,12 +114,10 @@ int FindArg(const char *which) {
 
   for (int i = 1; i <= TotalArgs; i++) {
     if (which_matches(GameArgs[i])) {
-      LOG_INFO.printf("FindArg: Found [%s] at argument index (%d).", which, i);
```

Comment:
I would like to retain logging supplied cmd-line options from user for diagnostic purposes. This may useful to identify non-standard user environments.

Reply:
With the check for "-nosound" I introduced it was spamming logging non-stop, it's a quick and dirty way to make it stop. I'll do it more properly using a global cache variable to disable sound, no worries